config-example.js

```diff
@@ -3,9 +3,14 @@ var slackbot = require('./lib/bot');
 var config = {
 
   showIssueDetails: true,
-  issueDetailsToShow: {'fields.summary':1 , 'fields.assignee' : 1, 'fields.creator' : 0, 'fields.description': 0},
-  showDetailsByDefault: true,//if true, you don't need the '+' to get details
+  basicIssueDetailsToShow: {'fields.summary':0 , 'fields.assignee' : 1, 'fields.creator' : 0, 'fields.description': 0},
+  fullIssueDetailsToShow: {'fields.summary':1 , 'fields.assignee' : 1, 'fields.creator' : 1, 'fields.description': 1},
+  showDetailsByDefault: true, //if true, you don't need the '+' to get details - instead use a '-' for fewer details
+  issueSeparator: "\n",
+  maxFieldLength: 100,
+  issueLabelSeparator: ": ",
   bot_name: "jira",//Provide the name to post under
+  bot_id: "jira",//Provide the name to post under
```

Comment:
I found with our setup, that whenever our bot (in this case it was @jirabot) was mentioned, the plugin was receiving a message that started with <SOMEID> instead. This was to cover that eventuality.